DependencyInjection/LiipImagineExtension.php

```diff
@@ -101,6 +102,17 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('templating.xml');
         }
 
+        $driver = $config['driver'];
+        if ($driver === 'vips') {
```

Comment:
```suggestion
        if ('vips' === $config['driver']) {
```

we could inline the variable here

Reply:
I use it in three other places instead.

Reply:
ah, i see. yeah lets keep it then.